sentry_sdk/serializer.py

```diff
@@ -138,23 +139,7 @@ def _should_repr_strings():
             are not (yet) in frame variables, but might encounter them when
             recursing (e.g.  we're in `event.exception`)
         """
-        try:
-            p0 = path[0]
-            if p0 == "stacktrace" and path[1] == "frames" and path[3] == "vars":
-                return True
-
-            if (
-                p0 in ("threads", "exception")
-                and path[1] == "values"
-                and path[3] == "stacktrace"
-                and path[4] == "frames"
-                and path[6] == "vars"
-            ):
-                return True
-        except IndexError:
-            return None
-
-        return False
+        return is_vars
```

Comment:
Can't comment on the specific lines because not part of the diff, but:
L123: return type is now always bool
Please also update the docstring

Reply:
removed method and documented `serialize` instead.